common/metrics/service_wrapper.go

```diff
@@ -45,24 +45,24 @@ type (
 )
 
 const (
-	scopeNameDeprecateDomain                = "DeprecateDomain"
```

Comment:
Check https://github.com/uber-go/tally#report-your-metrics

It is possible to add a prefix when creating the tally scope. That would be a cleaner approach than adding this string everywhere.

Reply:
I tried to use subscope, it is not as easy as it sounds. The way our code is implemented, the metrics scope is shared by our library code and the user code. If we want to use subscope, we would need to create a new filed for multiple places to hold the original user's metrics and the subscope. I would rather not do in this change and maybe revisit if we want to make that change later.
Added constant for the prefix definition to avoid putting string everywhere.